sys/arm64/include/_align.h

```diff
@@ -38,7 +38,7 @@
  * for all data types (int, long, ...).   The result is unsigned int
```

Comment:
Who wrote this comment... it's a size_t which is definitely not an unsigned int on arm64... presumably inherited from the arm port...

Reply:
It's been copy-pasted from i386 to all the others it seems.  Originated in pre-FreeBSD:

https://svnweb.freebsd.org/csrg?view=revision&revision=50141

That comment may be older than some of you...

Reply:
The VAX version was a bit older:

https://svnweb.freebsd.org/csrg?view=revision&revision=40721

Reply:
By nearly 5y in fact! Is it really that much to ask for people to review the thing they copy/paste...